src/Common/src/Interop/Unix/System.Native/Interop.InitializeSignalHandling.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        private static bool s_signalHandlingInitialized = false;
```

Comment:
Right!

I am wondering: is it supposed to work that a single Unix process may host .NET Core several times?
I guess those instances would share the global variables in the native implementation?

Reply:
> is it supposed to work that a single Unix process may host .NET Core several times?

We do not support or test config like these. It should be possible in theory, but I doubt that it would "just work".

> I guess those instances would share the global variables in the native implementation?

Right.